<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addone</class>
 <widget class="QDialog" name="addone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#addone{background-image: url(:/new/myimage/new_icon/Background.jpg);}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>430</width>
     <height>611</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>430</width>
     <height>500</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 203, 216);
font: 18pt &quot;华文楷体&quot;;

</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>收入</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>411</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="icon_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="icon_5">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="icon_7">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="income">
        <property name="text">
         <string>其它</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="borrowedfund">
        <property name="text">
         <string>借入</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="icon_1">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="parttimejob">
        <property name="text">
         <string>兼职</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="subsidy">
        <property name="text">
         <string>补贴</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="lottery">
        <property name="text">
         <string>彩票</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="salary">
        <property name="text">
         <string>工资</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="bonus">
        <property name="text">
         <string>奖金</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="icon_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="icon_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="icon_6">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>类型：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>金额：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="moneybox_income">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>元</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>日期：</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdi_income">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save_income">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>490</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color:#000000;
	border:2px solid #ffffff;
	background-color:#bbcbd8;
	border-radius:5px;
}

QPushButton::hover{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::pressed,QPushButton::checked{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::disabled{
	background-color:transparent;
}
</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel_income">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>530</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color:#000000;
	border:2px solid #ffffff;
	background-color:#bbcbd8;
	border-radius:5px;
}

QPushButton::hover{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::pressed,QPushButton::checked{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::disabled{
	background-color:transparent;
}
</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="in_remark_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>411</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>支出</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>类型：</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>411</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QRadioButton" name="goout">
        <property name="text">
         <string>出行</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="icon_14">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="icon_12">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="eating">
        <property name="text">
         <string>餐饮</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="icon_8">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="care">
        <property name="text">
         <string>医疗</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="outcome">
        <property name="text">
         <string>其它</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="education">
        <property name="text">
         <string>教育</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="icon_10">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="fun">
        <property name="text">
         <string>娱乐</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>购物</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="icon_9">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="icon_13">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="icon_11">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDateEdit" name="dateEdit_outcome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>金额：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="moneybox_outcome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>日期：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel_outcome">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>530</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color:#000000;
	border:2px solid #ffffff;
	background-color:#bbcbd8;
	border-radius:5px;
}

QPushButton::hover{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::pressed,QPushButton::checked{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::disabled{
	background-color:transparent;
}
</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save_outcome">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>490</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	color:#000000;
	border:2px solid #ffffff;
	background-color:#bbcbd8;
	border-radius:5px;
}

QPushButton::hover{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::pressed,QPushButton::checked{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::disabled{
	background-color:transparent;
}
</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>元</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="out_remark_textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>411</width>
       <height>64</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文楷体</family>
       <pointsize>18</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
